mammoth/style_reader/document_matcher_reader.py

```diff
@@ -90,6 +92,10 @@ def _read_list_node(list_node):
 list = ":" list_type "(" ~"[0-9]+" ")"
 
 list_type = "ordered-list" / "unordered-list"
+
+line_break = "br" line_break_type?
+
+line_break_type = "page" / "column"
```

Comment:
These changes to the grammar and parser don't seem to be tested anywhere, nor do they actually seem to be used anywhere (as mentioned in my comment in `conversion.py`).